.travis.yml

```diff
@@ -0,0 +1,46 @@
+# Run only on PRs and the master branch
+branches:
+  only:
+    - master
+
+# Default to use the container infrastructure
+dist: trusty
+sudo: false
+
+# Chttps://docs.travis-ci.com/user/caching/#Rust-Cargo-cache
+cache:
+  - cargo
+  - bundler
+
+# Set the default language to rust
+# https://docs.travis-ci.com/user/languages/rust/
+language: rust
+rust:
+  - 1.15.0    # What we ship now
+  - stable    # What we could ship in the next release
+  - beta      # What we'll be using in a few months (likely)
+
+# Since the delivery-cli ships in the ChefDK, we use the ChefDK as a base
+# ruby to ensure everything works appropriately.
+addons:
+  apt:
+    sources:
+      - chef-stable-precise
+    packages:
+      - chefdk
+install:
+  - eval "$(/opt/chefdk/bin/chef shell-init bash)"
+
+# Delivery CLI requires Git to be configured
+before_script:
+  - git config --global user.name "Chef CI"
```

Comment:
IMHO we could use environment variables. And even better adding them in the `Makefile` instead of the `.travis.yml`